.github/ISSUE_TEMPLATE.md

```diff
@@ -1,41 +1,19 @@
-Please read the following information before posting the issue.
-
-## Before posting the issue...
-
-* If you're having trouble with the NuGet.org Website, please post in [NuGetGallery issues](http://github.com/nuget/nugetgallery/issues)
-* If you're having trouble with the NuGet client tools (the Visual Studio extension, NuGet.exe command line tool, etc.), you are in the right place.
-
-Remove the content above here and fill out details below.
+<!-- Please read the following information before posting the issue. -->
+<!-- If you're having trouble with the NuGet client tools (the Visual Studio extension, NuGet.exe command line tool, etc.), you are in the right place. -->
+<!-- If you're having trouble with the NuGet.org Website, please post in [NuGetGallery issues](http://github.com/nuget/nugetgallery/issues) -->
 
 ## Details about Problem
 
-NuGet product used (NuGet.exe | VS UI | Package Manager Console | dotnet.exe):
-
-NuGet version (x.x.x.xxx):
-
-dotnet.exe --version (if appropriate):
-
-VS version (if appropriate):
+NuGet product used (NuGet.exe | VS | MSBuild | dotnet.exe):
 
-OS version (i.e. win10 v1607 (14393.321)):
+Product version: 
 
-Worked before? If so, with which NuGet version:
+Is this a regression? Which version: 
```

Comment:
Should we be more explicit in asking which version last worked?

Reply:
Any suggestions?
The MO for this change is reducing verbosity and unnecessary clutter, so I wanted to make it shorter. 

To me, this reads just fine, but open to suggestions.